### PR TITLE
Update to kotlin 1.9

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Update to kotlin 1.9

### CODE REVIEW

**Review by kgevorkyan — COMMENTED** [review]
postponed for now

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [versions]
-kotlin = "1.8.22"
+kotlin = "1.9.0"
```
Comment (nulls) [inline gradle/libs.versions.toml:2]:
```suggestion
kotlin = "1.9.10"
```

Comment (kgevorkyan) [reply gradle/libs.versions.toml:2]: anyway its blocked for now

**Comment by nulls** [comment]
@akuleshov7

**Review by akuleshov7 — APPROVED** [review]

**Comment by akuleshov7** [comment]
> @akuleshov7, it's required for save-cloud migration to 1.9

Go, bro

**Review by nulls — APPROVED** [review]

**Comment by nulls** [comment]
`com.saveourtool.save.plugins.fixandwarn.FixAndWarnPluginTest#base test` is flaky test on Windows is due to:
`The process cannot access the file because it is being used by another process.`

**Pull request merged**